app/routes/notices-setup.routes.js

```diff
@@ -4,6 +4,8 @@ const NoticesSetupController = require('../controllers/notices-setup.controller.
 
 const basePath = '/notices/setup'
 
+const ABSTRACTION_ALERTS_PATH = 'abstraction-alerts'
```

Comment:
Is SonarQube shouting at us about this? Because if not, in the routes file, I think it is preferable to have the route stated explicitly.

We've not had to or done this for any of our other routes files.